main.py

```diff
@@ -1,5 +1,34 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Actor, Genre
+
 
 def main():
-    pass
+
+    # create genres and actors
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Kianu")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
```

Comment:
I think it's could be easier if you use a some kind of collection, and then create object in the cycle with arguments from this collection